### PR TITLE
Style and implementation of component avatar

### PR DESCRIPTION
**Pull request by jules-n**
Small component, that implements the functionality of user's image and "mini avatar"

### CODE REVIEW

**Review by ChernykhVyacheslav — APPROVED** [review]

File: src/components/ProfilePage/Avatar/Avatar.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import './avatar.css';
+import React from 'react';
+
+const Avatar = React.forwardRef((props, ref) => {
+    return (
+        <img className="img"  ref = {ref}onClick={props.onClick} src={props.src} width={props.width} alt="user's avatar" accept=".jpg, .jpeg, .png"></img>
```
Comment (achehrynets) [inline src/components/ProfilePage/Avatar/Avatar.js:6]: Fix code style (formatting):
`ref = {ref}onClick={props.onClick}`

Comment (achehrynets) [reply src/components/ProfilePage/Avatar/Avatar.js:6]: I think it is better to capitalize the sentence for a user:
`alt="user's avatar"`

File: src/components/ProfilePage/Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.img{
```
Comment (achehrynets) [inline src/components/ProfilePage/Avatar/avatar.css:1]: Fix code style:
`.img{`

**Review by achehrynets — CHANGES_REQUESTED** [review]
Looks good, except for few code style issues.

File: src/components/ProfilePage/Avatar/Avatar.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import './avatar.css';
+import React from 'react';
+
+const Avatar = React.forwardRef((props, ref) => {
+    return (
+        <img className="avatar"  ref={ref} onClick={props.onClick} src={props.src} width={props.width} alt="User's avatar" accept=".jpg, .jpeg, .png"></img>
+    )
+    })
```
Comment (achehrynets) [inline src/components/ProfilePage/Avatar/Avatar.js:8]: Fix code style issue.

File: src/components/ProfilePage/Avatar/avatar.css [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.avatar{
```
Comment (achehrynets) [inline src/components/ProfilePage/Avatar/avatar.css:1]: Fix code style issue.

**Review by achehrynets — CHANGES_REQUESTED** [review]
Requested changes.

**Review by nelfan — APPROVED** [review]
+1

**Review by ChernykhVyacheslav — APPROVED** [review]

**Pull request merged**